h/api/models/document.py

```diff
@@ -182,6 +186,153 @@ def __repr__(self):
         return '<DocumentMeta %s>' % self.id
 
 
+def create_or_update_document_uri(db,
```

Comment:
@nickstenning mentioned in one of my previous pull requests that when passing in the database connection, we should call the parameter `session`.
The comment is [here](https://github.com/hypothesis/h/pull/3011#discussion_r54537119)